demos/fleet_provisioning/fleet_provisioning_with_csr/fleet_provisioning_with_csr_demo.c

```diff
@@ -60,16 +60,19 @@
 /* Demo config. */
 #include "demo_config.h"
 
-/* MQTT operations. */
-#include "mqtt_operations.h"
-
 /* TinyCBOR library for CBOR encoding and decoding operations. */
 #include "cbor.h"
 
+/* corePKCS11 includes. */
+#include "core_pkcs11.h"
+#include "core_pkcs11_config.h"
```

Comment:
The library-specific config files shouldn't be necessary to be included in the application as they are meant to be included in the library itself. (The config file gets indirectly included in the application through the library header). 
```suggestion
#include "core_pkcs11.h"
```

Reply:
This is false. The config file is not indirectly included.

Reply:
Interesting, that is different for corePKCS11 than other libraries.